docs/README.md

```diff
@@ -1,50 +1,98 @@
+    doxygen Doxyfile
 
-    cmake-gui
+in the `docs/` directory.
 
-Set the `WITH_DOCS` flag to true in the CMake GUI, then hit configure + generate.
-Build, successively, the doxygen project and then the sphinx project.
 
-**NOTE:** The sphinx project will fail its build if doxygen isn't built first.
-If you do not, the Docs section (that uses doxygen) will display a warning of the form:
+## Read the Docs
 
-```
-!Warning
-oxygenclass: breathe_default_project value ‘OmniPhotos’ does not seem to be a valid key for the breathe_projects dictionary
-```
+Read the Docs virtual machines read the [sphinx](https://www.sphinx-doc.org/) `conf.py` file and generate the documentation accordingly.
+We have injected the call to doxygen into this file.
+The doxygen binary is then executed in the docs directory and generates files in the `docs/html/` directory, replacing the generated sphinx html files with those build by doxygen.
 
-The Doxyfile.in is where you want to make your modifications to the html/xml output from Doxygen.
-This will be copied over and filled in by CMake at build time.
-To switch this on or off the html generation for doxygen set the `GENERATE_HTML` flag in `Doxyfile.in` and run the configure + generate pipeline again.
-There is also a `HIDE_UNDOC_CLASSES` that will reduce the build time if set to "yes".
-If you want the standard doxygen HTML files with all the classes, then set this flag to "no" as well as the `GENERATE_HTML` flag to "yes".
+The latest generated code documentation is available at https://omniphotos.readthedocs.io/en/latest/.
 
-To run the Doxygen section of the build on its own, run the following command in 
-the `build/docs` directory once you've run CMake:
 
-    doxygen ..build/docs/doxygen/Doxyfile
+# Writing Documentation
 
-In order to run the sphinx build step independently (once you've run the doxygen step successfully), run the following command from the `docs` directory:
-    
-    sphinx-build -E -b html -Dbreathe_projects.OmniPhotos=../build/docs/doxygen/xml . ../build/docs/sphinx
+## Getting started
+
+In order to create a doxygen code stump, one can use the **Doxygen Comments** extension found in the Visual Studio online extension manager.
+It is not necessary to use this, but it seems to be the easiest option:
+
+<img src='images/vs_screenshot_1.png'/>
+
+The extension will generate a stub whenever you type `/**` and press enter above a function, variable or class.
+
+The stumps can be customized via Tools->Options and selecting Doxygen in the left-hand pane.
+
+**Other Option:**
+Use the Visual Studio Plugin **1-Click Docs**.
+You can use the **1-Click Docs** extension to call `doxygen.exe` in Visual Studio.
+For more information please refer to https://www.doxygen.nl/manual/starting.html.
+
+There is a demo project (https://github.com/yuanmingze/codestyle_demo) for Doxygen use.
+For a tutorial of how to use Doxygen in Visual Studio 2019, please reference the project's README.md file.
```

Comment:
What does this line refer to? @yuanmingze @reubenlindroos 

It would be good to either clarify or remove this line.

Reply:
@cr333 do you mean line 45?

Reply:
I was referring to line 48. Could you please clarify or remove it?

Apologies for the confusion.